test/features/install_explainability.feature

```diff
@@ -0,0 +1,56 @@
+@explainability
+@kafka
+Feature: Kogito Explainability service
+
+  Background:
+    Given Namespace is created
+
+  @smoke
+  # Disabled as long as https://issues.redhat.com/browse/KOGITO-3176 is not solved
+  @disabled
+  Scenario: Install Kogito Explainability
+    Given Kogito Operator is deployed with Kafka operator
+    And Install Kogito Explainability with 1 replicas
+    Then Kogito Explainability has 1 pods running within 10 minutes
+    And HTTP GET request on service "explainability" is successful within 2 minutes with path "health/live"
+
+#####
+
+  @events
+  @kafka
+  Scenario: Explainability retrieves explainability requests events using external Kafka
```

Comment:
In this scenario, we're testing this integration using external kafka and infinispan instances. Usually, we tag these scenarios using the annotation "@externalcomponent" (see example [here](https://github.com/kiegroup/kogito-cloud-operator/blob/3fa1d80556ac5d89f26b4be17145196576863f19/test/features/operator/install_jobs_service.feature#L78)).

However, I'm also missing a scenario using the kafka and Infinispan instances that are deployed automatically.